_commands/date.md

```diff
@@ -0,0 +1,29 @@
+---
+---
+
+date
+-------
+
+The date command is used to print out, or change the value of, the system's time and date information.
+
+~~~ bash
+date [OPTION] ... [+FORMAT]
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### date [-u|--utc|--universal] [MMDDhhmm[[CC]YY][.ss]]
+
+##### OPTION:  -u, --utc, --universal 	 Print or set Coordinated Universal Time.
```

Comment:
fix formatting, make sure code appears as code and body text isn't being formatted as a header